src/components/Route/CreateRoute.tsx

```diff
@@ -0,0 +1,195 @@
+import { Button, FormControl, Input, Text, VStack, Select, HStack } from 'native-base';
+import { useState } from 'react';
+import { createRoute } from '../../xplat/api';
+import Popup from 'reactjs-popup';
+import 'reactjs-popup/dist/index.css';
+import { User } from '../../xplat/types/user';
+import { RouteType, RouteClassifier, RouteColor, RouteTech } from '../../xplat/types/route';
+import { getAllBoulderClassifiers, getAllTraverseRouteClassifiers, 
+  getAllCompetitionRouteClassifiers,
+  getAllLeadclimbRouteClassifiers,
+  getAllTopropeRouteClassifiers } from '../../xplat/api';
+
+
+const RouteTypeToGetAllClassifiers = (type: RouteType) => {
+  if (type === RouteType.Boulder)
+    return getAllBoulderClassifiers();
+  if (type === RouteType.Traverse)
+    return getAllTraverseRouteClassifiers();
+  if (type === RouteType.Competition)
+    return getAllCompetitionRouteClassifiers();
+  if (type === RouteType.Leadclimb)
+    return getAllLeadclimbRouteClassifiers();
+  return getAllTopropeRouteClassifiers();
+};
+
+const Ropes = [
+  '1', '2', '3', '4', '5', '6', '7', '8', '9'
+];
```

Comment:
Should this be in xplat? Lmao

Reply:
Maybe? If mobile will use it for any kind of drop-down then yeah but I think they'll just be seeing ropes when fetching a route.